integration/nemo/examples/supervised_fine_tuning/README.md

```diff
@@ -0,0 +1,190 @@
+## Supervised Fine-tuning (SFT) with NeMo
+
+In this example, we utilize NeMo's [supervised fine-tuning](https://github.com/NVIDIA/NeMo-Megatron-Launcher#515-instruction-following-via-supervised-finetuning--sft-)
+feature to showcase how to fine-tune the whole model on supervised data for learning how to follow user specified instructions. 
+
+Due to the large model size of the LLM, we use NVFlare's streaming feature to transfer the model in chunks.
+
+## Dependencies
+This example running a 1.3B GPT model requires considerable computational resources. For training 1.3B model, SFT needs ~30GB GPU memory using fp16 precision. Hence, to run three clients in parallel, we can compute the resource needs accordingly.
+The example for 3-client 1.3B GPT model was performed on two A100 GPUs with 80 GB memory each.
```

Comment:
I tested with each client running on a 32gb V100. 24gb seems to be the minimum requirement (not tested).

Reply:
updated the info